bumble/profiles/asha_service.py

```diff
@@ -76,8 +76,10 @@ def on_audio_control_point_write(_connection, value):
                     f'volume={value[3]}, '
                     f'otherstate={value[4]}'
                 )
+                self.emit('start', str(value[0]).encode())
             elif opcode == AshaService.OPCODE_STOP:
                 logger.info('### STOP')
+                self.emit('stop', str(value[0]).encode())
```

Comment:
`stop` doesn't have any arguments (and value[0] is the opcode, it doesn't seem to be useful to emit the `stop` opcode as an argument to the `stop` event.

Reply:
ack